Reviewer/src/main/java/com/softserveinc/reviewer/ReviewerApplication.java

```diff
@@ -22,7 +22,9 @@ public String getName() {
     @Override
     public void run(final ReviewerConfiguration configuration, final Environment environment) {
         Injector injector = Guice.createInjector(new GuiceModule(configuration));
-        environment.healthChecks().register("ReviewerHealthCheck", injector.getInstance(ReviewerHealthCheck.class));
+        environment.healthChecks().register("SyndicationHealthCheck", new DataProvidersHealthCheck(configuration.getSyndicationHealthCheckUrl()));
+        environment.healthChecks().register("OracleHealthCheck", new DataProvidersHealthCheck(configuration.getOracleHealthCheckUrl()));
+        environment.healthChecks().register("ElasticSearchHealthCheck", new DataProvidersHealthCheck(configuration.getElasticSearchHealthCheckUrl()));
```

Comment:
Please change it to be switchboard, oracle and elasticsearch respectively. No need to call it health check explicitly.